includes/wikia/GlobalFunctions.php

```diff
@@ -104,22 +104,14 @@ function print_pre( $param, $return = 0 )
  *
  * @author Inez Korczyński <inez@wikia-inc.com>
  *
- * @param String $url -- old url
- * @param String $timestamp -- last change timestamp
+ * @param string $url -- old url
+ * @param string|false $timestamp -- last change timestamp
  *
- * @return String -- new url
+ * @return string -- new url
  */
 function wfReplaceImageServer( $url, $timestamp = false ) {
 	$wg = F::app()->wg;
 
-	// Override image server location for Wikia development environment
-	// This setting should be images.developerName.wikia-dev.com or perhaps "localhost"
-	// FIXME: This needs to be removed. It should be encapsulated in the URL generation.
-	$overrideServer = !empty( $wg->DevBoxImageServerOverride ) && !$wg->EnableVignette;
-	if ( $overrideServer ) {
-		$url = preg_replace( "/\\/\\/(.*?)wikia-dev\\.(pl|us|com)\\/(.*)/", "//{$wg->DevBoxImageServerOverride}/$2", $url );
-	}
-
 	wfDebug( __METHOD__ . ": requested url $url\n" );
 	if ( substr( strtolower( $url ), -4 ) != '.ogg' && isset( $wg->ImagesServers ) && is_int( $wg->ImagesServers ) ) {
 		if ( strlen( $url ) > 7 && substr( $url, 0, 7 ) == 'http://' ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)